.evergreen/config.yml

```diff
@@ -2121,19 +2121,19 @@ axes:
           - windows-64-vs2017-test
         variables:
           GCC_PATH: "/cygdrive/c/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin"
-          GO_DIST: "C:\\golang\\go1.18"
+          GO_DIST: "C:\\golang\\go1.19"
           VENV_BIN_DIR: "Scripts"
       - id: "ubuntu1604-64-go-1-18"
```

Comment:
The id values should updated, correct? i.e. "ubuntu1604-64-go-1-19"

I think this also requires updating the matrices, [here](https://github.com/mongodb/mongo-go-driver/blob/c9b3eb092798b2016dc3a7b8083ef3f707320bd9/.evergreen/config.yml#L2412).

Reply:
Then documentation [here](https://github.com/mongodb/mongo-go-driver/blame/master/README.md#L26) should also be updated.

Reply:
I think the labels are entirely descriptive. I don't have a strong opinion, I suppose the argument for maintaining the Go Version is that it illustrates which version of Go we're testing on without having to visit the evergreen config file or read testing logs.

Reply:
Sounds good, I'll keep it and update it.